provider/clusterrosa/classic/cluster_rosa_classic_resource.go

```diff
@@ -475,7 +485,7 @@ func createClassicClusterObject(ctx context.Context,
 	builder := ocmClusterResource.GetClusterBuilder()
 	clusterName := state.Name.ValueString()
 	if len(clusterName) > rosa.MaxClusterNameLength {
-		errDescription := fmt.Sprintf("Expected a valid value for 'name' maximum of 15 characters in length. Provided Cluster name '%s' is of length '%d'",
+		errDescription := fmt.Sprintf("Expected a valid value for 'name' maximum of 54 characters in length. Provided Cluster name '%s' is of length '%d'",
```

Comment:
```suggestion
		errDescription := fmt.Sprintf("Expected a valid value for 'name' maximum of %d characters in length. Provided Cluster name '%s' is of length '%d'", rosa.MaxClusterNameLength
```

Just for future proofing